src/cnpj/validation.ts

```diff
@@ -0,0 +1,70 @@
+export function validateCNPJ(cnpjReceveid: string | number) {
+  let cnpj = String(cnpjReceveid).replace(/[&\/\\#,+()$~%.'":*?<>{}-]/g, '')
+
+  let cnpjArray = cnpj
+    .replace(/[^\d]+/g, '')
+    .trim()
+    .split('')
+    .map((char: any) => Number(char))
```

Comment:
why use the `any` type when the param type can be inferred by ts automatically?
```suggestion
    .map((char) => Number(char))
```